### PR TITLE
React new

### PR DESCRIPTION
**Pull request by wabowen89**

### CODE REVIEW

File: src/pages/New.js [hunk 1]
```diff
@@ -1,10 +1,112 @@
...
+
+const New = ({ createTask, currentUser }) => {
+  const [newTask, setNewTask] = useState({
+    Area: "",
+    Item: "",
+    Image: "",
+    Task: "",
+    Description: "",
+    Frequency: "",
+    Due_date: "",
```
Comment (SunkissedQueen) [inline src/pages/New.js:16]: attributes should be lowercase

File: src/_tests_/New.test.js [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+      <BrowserRouter>
+        <New createTask={jest.fn()} currentUser={{ id: 1 }} />
+      </BrowserRouter>
+    );
+
+    fireEvent.click(screen.getByText('Cancel'));
+
+    expect(navigate).toHaveBeenCalledWith('/index');
+  });
 });
```
Comment (SunkissedQueen) [inline src/_tests_/New.test.js:25]: good approach to mock function

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
I feel there might be some merge conflicts since files were changed on this branch that were also changed on a different branch. Similar questions from the other branch:
Curious what is the difference between the current PascalCase react styling components and the components I see with the . separating each word. Ex: <Nav.Item></Nav.Item> and <NavItem></NavItem>?
Please look over the files for uniform indentation, proper nesting, and commented code that you are not using.
I see that the components are accepting a lot of props. I did not notice where the data is being stored. Where are these props being passed?

**Comment by kyloh1996** [comment]
I feel there might be some merge conflicts since files were changed on this branch that were also changed on a different branch

There was a merge conflict at one point but we handled it :)

File: src/pages/New.js (hunk 1 above)
Comment (kyloh1996) [reply src/pages/New.js:16]: attributed have been downsized :P

File: src/pages/NotFound.js [hunk 3]
```diff
@@ -1,10 +1,26 @@
...
+    </div>
+    <div className='returnBtn'>
+    <button style={{backgroundColor: '#992509', height:'6vw', width:'20vw', borderRadius:'2vw', color: '#ffffff',}} onClick={handleReturnHome}>Return to the Hive</button>
+    </div>
+    <div className='cutePics'>
+    <img src='../images/beeHive.jpeg' alt='bee' className="pic1"/>
+    <img src='../images/bee1.jpeg' alt='bee' className='pic2' />
+    </div>
+    </div>
   );
```
Comment (SunkissedQueen) [inline src/pages/NotFound.js:24]: indentation

File: src/styles/Home.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    height: 27vw !important;
+    background-color: #dc9427 !important;
+    
+  }
+  
+.welcome{
+font-family: 'Eczar SemiBold';
+font-size: 4vw;
+padding: 3vw;
+}
```
Comment (SunkissedQueen) [inline src/styles/Home.css:27]: indentation

File: src/styles/NotFound.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+height: 25vw;
+
+
+}
+.pic2{
+width: 30vw;
+margin-bottom: 43%;
+z-index: 1;
+margin-left: 23vw;
+}
```
Comment (SunkissedQueen) [inline src/styles/NotFound.css:33]: indentation, commented out code

**Review by SunkissedQueen — APPROVED** [review]
Looks like there will be some merge conflict

**Pull request merged**